core/components/Footer/Footer.tsx

```diff
@@ -13,8 +13,13 @@ const Footer = () => {
   };
 
   return (
-    <BottomSection data-testid="footer-box" onClick={handleResetProgress}>
+    <BottomSection
+      data-testid="footer-box"
+      onClick={handleResetProgress}
+      data-cy="cy-footer"
```

Comment:
no need `data-testid` and `data-cy` please use only `data-testid`, for example:
`cy.findByTestId('footer-box')`

Reply:
This function does not exists on cy

Reply:
It should found we use it:
![image](https://user-images.githubusercontent.com/40696945/182846079-e3ff4769-5d6a-4ca0-a053-7c61fbf66a70.png)